mod/notifications.php

```diff
@@ -79,13 +78,14 @@ function notifications_content(App $a)
 	}
 
 	$page = ($_REQUEST['page'] ?? 0) ?: 1;
-	$show =  $_REQUEST['show'] ?? 0;
+	$show =  isset($_REQUEST['show']) ? $_REQUEST['show'] : false;
```

Comment:
Hum, this wasn't meant to be the expected replacement, just to show you what parentheses or a lack thereof can do to a null coalescing operator. If this is what you intended, it would be better as 
```suggestion
	$show = boolval($_REQUEST['show'] ?? false);
```

Reply:
Ah .. OK .. I reverted it to the removed parenthess version since better readability